CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   * `ruler.storage.type` has beem added to specify the rule store backend type, currently only the configdb.
   * `ruler.poll-interval` has been added to specify the interval in which to poll new rule groups.
 * [FEATURE] The distributor can now drop labels from samples (similar to the removal of the replica label for HA ingestion) per user via the `distributor.drop-label` flag. #1726
+* [FEATURE] The Querier can now send queries to Ingesters only if the query fits within MaxLookback
```

Comment:
A couple of things:
1. Please add the PR number
2. `MaxLookback` looks unclear to me. We should mention the specific config option
3. I would also mention the new flag/config option introduced to enable it

Reply:
Fixed